earth_enterprise/tutorial/GoogleEarthFusionTutorial.spec

```diff
@@ -1,49 +0,0 @@
-%define _prefix /opt/google/share/tutorials/fusion
```

Comment:
I suppose I can add a README.md file with a link to download the files if you want to create the RPM.